TELCODOCS-806-1 PR for setting interface sysctl with SR-IOV (#49871)
openshift/openshift-docs: modules/nw-sriov-interface-level-sysctl-basic.adoc

// Module included in the following assemblies:
//
// * networking/hardware_networks/configuring-interface-sysctl-sriov-device.adoc

:_content-type: CONCEPT
[id="nw-basic-example-setting-one-sysctl-flag_{context}"]

> = Basic example setting one sysctl flag

Reviewer: ```suggestion
= Setting one sysctl flag
```

> As a cluster administrator, you can set interface-level network sysctls for a pod connected to a SR-IOV network device.

Reviewer: ```suggestion
You can set interface-level network `sysctl` settings for a pod connected to a SR-IOV network device.
```

Author: done

> In this example the `net.ipv4.conf.IFNAME.accept_redirects` is set to `1` on the created virtual interfaces.

Reviewer: ```suggestion
In this example, `net.ipv4.conf.IFNAME.accept_redirects` is set to `1` on the created virtual interfaces.
```

Author: done
